prism/static/js/src/studio.js

```diff
@@ -1,6 +1,7 @@
 /* Javascript for PrismXBlock. */
-function PrismXBlockEditBlock(runtime, element) {
-    var $element = $(element);
+function PrismXBlock(runtime, element) {
+
+    console.log(runtime);
 
     var lngSelect = document.getElementById('language');
     var thmSelect = document.getElementById('theme');
```

Comment:
Out of scope, I know. But it's better to use prefix to avoid collision.

```suggestion
    var lngSelect = $(element).find('#language');
    var thmSelect = $(element).find('#theme');
```

If you'd like to accept this change, please update the lines below as well:

```js
var data = {
            display_name: $(element).find('input[name=display_name]').val(),
            code_data: codeMirror.getValue(),
            language: lngSelect.options[lngSelect.selectedIndex].value,
            theme: thmSelect.options[thmSelect.selectedIndex].value,
            maxheight: $(element).find('input[name=maxheight]').val(),
        };
```